exonum_client/client.py

```diff
@@ -17,6 +17,7 @@
 # Example of a formatted prefix: "https://127.0.0.1:8000"
 _ENDPOINT_PREFIX = "{}://{}:{}"
 
+_PROTO_SOURCES_URL = _ENDPOINT_PREFIX + "/api/runtimes/rust/{}"
```

Comment:
Nit: This URL template is not directly corresponds to the `proto-sources` endpoint. Maybe name it something like `_RUST_RUNTIME_URL`?

Reply:
Agree. I forgot to rename it after i'd made some changes